moonscript/base.moon

```diff
@@ -34,6 +34,25 @@ to_lua = (text, options={}) ->
 
   code, ltable
 
+loadstring = (...) ->
+  options, str, chunk_name, mode, env = get_options ...
+  chunk_name or= "=(moonscript.loadstring)"
+
+  if string.sub(chunk_name,-8)==".litmoon"
```

Comment:
because honestly I have no idea how to decide it else. having the user decide by having 2 different functions seemed wrong because then what about `require`, etc?